Ginger/GingerCoreNET/RunLib/CLILib/CLIHelper.cs

```diff
@@ -579,5 +583,64 @@ internal void SaveAndCommitSelfHealingChanges()
                 Reporter.ToLog(eLogLevel.ERROR, "Failed to Check-in self healing changes in source control");
             }
         }
+
+        internal void SetSealightsEnable(bool? value)
+        {
+            if (value != null)
+            {
+                Reporter.ToLog(eLogLevel.DEBUG, "Selected SealightsEnable: '" + value + "'");
+                SealightsEnable = (bool)value;
+            }
+        }
+        internal void SetSealightsAgentToken(string value)
+        {
+            if (!String.IsNullOrEmpty(value))
+            {
+                Reporter.ToLog(eLogLevel.DEBUG, "Selected SealightsAgentToken: '" + value + "'");
```

Comment:
Agent token value is secret and we should not print it in logs. please remove the value from log statement

Reply:
agent token value is no longer printed to our logs